src/stories/Forms/Form/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const FormElement = styled.form`
+  /* Your styles here */
+`;
+
+export interface FormProps {
+  children: React.ReactNode;
+  [x: string]: any;
```

Comment:
What is this?

Reply:
To allow additional props for inputs. There are so many attributes to account for when using inputs.